index.html

```diff
@@ -0,0 +1,234 @@
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                        <button type="button">See Project </button>
+                    </ul>
+                </div>
+            </section>
+        </section>
+
+            <section class="about-myself">
+                <section class="skill">
+                    <h5>
+                        About <br />Myself
+                    </h5>
+                    <p>
+                        Hello I’m a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.
+                    </p>
+                    
+                    <div class="social_media">
+                        <h4>LET'S CONNECT</h4>
+                        <ul>
+                            <li>
+                                <a href="#">
+                                    <img src="img/Vector4.png" alt="github picture">
+                                </a>
+                            </li>
+                        </ul>
+                        <ul>
+                            <li>
+                                <a href="#">
+                                    <img src="img/Vector2.png" alt="linkedin picture">
+                                    <img src="img/Vector3.png" alt="linked picture">
+                                </a>
+                            </li>
+                        </ul>
+                        <ul>
+                            <li>
+                                <a href="#">
+                                    <img src="img/Vector5.png" alt="peace symbol">
+                                </a>
+                            </li>
+                        </ul>
+                        <ul>
+                            <li>
+                                <a href="#">
+                                    <img src="img/Vector.png" alt="twitter picture">
+                                </a>
+                            </li>
+                        </ul>
+                        <ul>
+                            <li>
+                                <a href="#">
+                                    <img src="img/Vector1.png"alt="Gmail picture">
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+
+                    <button type="button" class="get">Get my resume</button>
+
+                    <ul>
+                        <li>
+                            <div class="languages-skills">
+                                <img src="img/Frame 7.png">
+                            </div>
+                        </li>
```

Comment:
- [ ] Kindly make sure to add the title `Languages` which must appear at top of the languages skills list. Please make sure to align it as the provided design.

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/219609188-9f4b75ab-3cd0-44f7-b6ba-9ef885b48d9a.png)

   - ### FIGMA DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/219609120-e6edd700-5cd0-4292-9697-d885757e6342.png)